packages/base/src/SearchBar/SearchBar-android.tsx

```diff
@@ -141,9 +143,20 @@ export class SearchBarAndroid extends Component<
         testID="RNE__SearchBar-wrapper"
         style={StyleSheet.flatten([
           {
-            backgroundColor: theme?.colors?.white,
+            backgroundColor: theme?.colors?.searchBg,
             paddingTop: 8,
             paddingBottom: 8,
+            shadowColor: '#000',
+            shadowOffset: {
+              width: 0,
+              height: 1,
+            },
+            shadowOpacity: 0.2,
+            shadowRadius: 1.41,
```

Comment:
don't think elevation/shadow is needed

Reply:
I was trying to design like this https://material.io/design/navigation/search.html#persistent-search

Reply:
ok I will remove that